components/LegacyAnimation/LegacyAnimation.js

```diff
@@ -2,12 +2,13 @@ import * as Styled from "./LegacyAnimation.styled";
 
 export default function LegacyAnimation({ color, opacity }) {
   return (
-    <Styled.Container $opacity={opacity}>
-      <Styled.BlobsContainer>
-        <Styled.Blob $duration={"20s"} color={color} />
-        {/* {/* <Styled.Blob duration={"40s"} color={color} /> */}
-        <Styled.Blob $duration={"60s"} color={color} />
-      </Styled.BlobsContainer>
-    </Styled.Container>
+    <>
+      <Styled.Container color={color} $opacity={opacity}>
+        <Styled.BlobsContainer>
+          <Styled.Blob $duration={"2s"} color={color} />
+          <Styled.Blob $duration={"6s"} color={color} />
+        </Styled.BlobsContainer>
+      </Styled.Container>
+    </>
```

Comment:
We don't need the React Fragment here...

```suggestion

      <Styled.Container color={color} $opacity={opacity}>
        <Styled.BlobsContainer>
          <Styled.Blob $duration={"2s"} color={color} />
          <Styled.Blob $duration={"6s"} color={color} />
        </Styled.BlobsContainer>
      </Styled.Container>

```